tm2/pkg/bft/wal/wal_test.go

```diff
@@ -262,21 +262,26 @@ func BenchmarkWalRead10KB(b *testing.B) {
 }
 
 func BenchmarkWalRead100KB(b *testing.B) {
+	b.Skip("TODO: benchmark failing")
```

Comment:
Can these be a separate issue, since they relate to the WAL?

Reply:
https://github.com/gnolang/gno/issues/910